content/applications/productivity/documents.rst

```diff
@@ -246,21 +242,24 @@ Upload <documents/upload>` files to it.
 
 .. _documents/upload:
 
-Upload by email
-^^^^^^^^^^^^^^^
+Files upload
+^^^^^^^^^^^^
+
+Toggle the :guilabel:`Allow upload` switch to allow users to upload files to your workspace.
 
-Select the :guilabel:`Download and Upload` option to enable users to upload their files to your
-workspace using an :guilabel:`Email Alias`. To create the email alias, enter its name in the
-:guilabel:`Email Alias` field. The :doc:`domain name <../general/email_communication/email_servers>`
-should be set by default, but you can modify it by clicking it.
+Tick the :guilabel:`Upload by Email` checkbox to enable users to upload documents sent by email in
+the workspace.
 
-The documents sent to this email alias are uploaded to the workspace using the chosen
-:ref:`tags <documents/tags>`.
+Enter an alias in the :guilabel:`Email Alias` field. All documents sent to this email address are
+uploaded to the workspace using the chosen tags.
 
 .. note::
+   - Your **email server** must be configured to be able to create an **email alias**. If it isn't
+     configured yet, click :guilabel:`Choose or Configure Email Servers` and configure the
+     :doc:`mail server <../../applications/general/email_communication/email_servers>`;
    - By default, the :guilabel:`Document Owner` is the person who uploads a file to a workspace, but
      you can select another user. You can also set a :guilabel:`Contact`, usually an external
-     person, such as a partner.
+     person, such as a partner;
```

Comment:
and these can get a period at the end since they're full sentences :slightly_smiling_face: